conan/tools/google/bazeldeps.py

```diff
@@ -121,16 +121,23 @@ def _relativize_path(p, base_path):
             linkopts.append('"-l{}"'.format(linkopt))
         linkopts = ', '.join(linkopts)
 
+
+        lib_dir = 'lib'
+        if len(cpp_info.libdirs) != 0:
+            lib_dir = _relativize_path(cpp_info.libdirs[0], package_folder)
+
+        shared_library = dependency.options.get_safe("shared")
```

Comment:
Not all dependencies might have options. 
```suggestion
        shared_library = dependency.options.get_safe("shared") if dependency.options else False
```

Reply:
Done